apps/percona/deployers/percona-installer.yaml

```diff
@@ -0,0 +1,14 @@
+---
+- hosts: localhost
+  connection: local
+
+  tasks:
+    - name: Deploying percona
```

Comment:
- Before proceeding w/ the app deployment, there is a need to check for the availability of the storage class and then proceed if available (or fail if not). Ideally, to keep things generic, the storage class should be taken/obtained from the litmus app deployer job ENV. Please refer these tasks : https://github.com/openebs/litmus/blob/master/tests/mysql/mysql_data_persistence/test.yaml#L16 & https://github.com/openebs/litmus/blob/master/tests/mysql/mysql_data_persistence/test.yaml#L39

- Also, the PVC is an important object to be recorded and passed on to the "chaos test" stage from the app deployment stage. It is good to keep that recorded. (https://github.com/openebs/litmus/blob/master/tests/mysql/mysql_data_persistence/test.yaml#L33)

This would mean having App deployment templates bundled with the test

Reply:
sure sir